### ISSUE TITLE
Access denied to pull images from cp.icr.io. But have access to us.icr.io

### ISSUE THREAD

**Issue by y0zg** [IBM-Blockchain/ansible-collection#330]
hi @sstone1 

I'm referring to this doc 

https://ibm-blockchain.github.io/ansible-collection/tutorials/installing.html 

https://cloud.ibm.com/docs/openshift?topic=openshift-registry#secret_entitled_software

to deploy IBP on k8s via ansible and having access denied error while pulling images 

Error:
```
failed to resolve reference "cp.icr.io/cp/ibp-crdwebhook:2.5.0-20200618-amd64": pull access denied
```

```
docker login cp.icr.io --username cp --password <HIDDEN>
WARNING! Using --password via the CLI is insecure. Use --password-stdin.
WARNING! Your password will be stored unencrypted in /home/teevi/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded


============
docker pull cp.icr.io/cp/ibp-crdwebhook:2.5.0-20200618-amd64
Error response from daemon: pull access denied for cp.icr.io/cp/ibp-crdwebhook, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
Entitlement key is taken from URL: https://myibm.ibm.com/products-services/containerlibrary

**Question**: should I purchase additionally any IBP entitlement key (I use a key from URL https://myibm.ibm.com/products-services/containerlibrary
) or do any extra steps in order to get access to `cp.icr.io/cp/ibp-crdwebhook:2.5.0-20200618-amd64`

I see for manually deployed IBPs we use container registry `us.icr.io`, not `cp.icr.io`

**Comment by sstone1** [comment]
@y0zg IBM Blockchain Platform comes in two forms, SaaS and software.

- The SaaS is available on IBM Cloud and is provisioned through the IBM Cloud catalog (https://cloud.ibm.com/catalog/services/blockchain-platform)
- The software is a set of Docker images that can be deployed into any Red Hat OpenShift or Kubernetes cluster, accessed through the IBM Container Registry.

The tutorial you reference is for use with the IBM Blockchain Platform software form only.

As you mention the use of `us.icr.io`, I think this means that you have actually been using the SaaS - is that correct?

The SaaS can be provisioned by anyone with an IBM Cloud account which is attached to a funding source (e.g. credit card). 

However, to use the software, you must purchase entitlement for the IBM Blockchain Platform software from an IBM sales representative. Have you purchased an entitlement for the IBM Blockchain Platform software?

If you want to use this Ansible Collection with the SaaS, then you must provision the SaaS instance yourself through the IBM Cloud catalog. This Ansible Collection can work with an existing SaaS instance, but it cannot create new SaaS instances for you.

**Comment by y0zg** [comment]
@sstone1 

>  I think this means that you have actually been using the SaaS - is that correct?

Yes, correct

> Have you purchased an entitlement for the IBM Blockchain Platform software?

I used entitlement from https://myibm.ibm.com/products-services/containerlibrary
but it sounds like we need another entitlement specially for IBM Blockchain Platform in order to download images.  It's a bit strange as I supposed any provisioning should be done via IBM console, so I used entitlement from IBM console available for me. Need to check with IBM representatives regarding this.

Thank you for clarifications!
